src/client/components/Inputs/TextArea/TextArea.tsx

```diff
@@ -1,31 +1,64 @@
 import './TextArea.scss'
 import React, { forwardRef, TextareaHTMLAttributes, useEffect, useImperativeHandle, useRef } from 'react'
 
-type Props = Pick<TextareaHTMLAttributes<HTMLTextAreaElement>, 'disabled' | 'onChange' | 'onPaste' | 'value'>
+type AdditionalProps = {
+  maxHeight?: number | null
+}
+
+type Props = Pick<
+  TextareaHTMLAttributes<HTMLTextAreaElement>,
+  'disabled' | 'onChange' | 'onPaste' | 'placeholder' | 'rows' | 'value'
+> &
+  AdditionalProps
 
 const TextArea = forwardRef<HTMLTextAreaElement, Props>((props, outerRef) => {
-  const { disabled, onChange, onPaste, value } = props
+  const { disabled, maxHeight, onChange, onPaste, placeholder, rows, value } = props
 
   const textAreaRef = useRef<HTMLTextAreaElement>(null)
   useImperativeHandle(outerRef, () => textAreaRef.current!, [])
 
-  useEffect(() => {
+  const adjustTextAreaHeight = (textAreaRef: React.MutableRefObject<HTMLTextAreaElement>, maxHeight: number | null) => {
     const textArea = textAreaRef.current
+    if (!textArea) return
+
     textArea.style.height = 'auto'
-    textArea.style.height = `${textArea.scrollHeight}px`
-  }, [disabled, textAreaRef, value])
+    const newHeight = Math.min(textArea.scrollHeight, maxHeight)
+    textArea.style.height = `${newHeight}px`
+  }
+
+  useEffect(() => {
+    const handleResize = () => {
```

Comment:
Yes, great suggestion. I added it with 200 ms because I also implemented enabling the scrolling and it was a little slow if the user tried to use it immediately.  

https://github.com/openforis/fra-platform/assets/41337901/103160ee-3a80-4fd6-943f-15faacc02b96